RedHat-UX/red-hat-design-system
# fix(cta): use flex gap and remove inline padding

## What I did

1. Removed inline padding from shadowdom svg and pf-icon, added a flex gap replacement.

Closes #833

## Testing Instructions

1. View [deploy preview demo for cta](https://deploy-preview-835--red-hat-design-system.netlify.app/components/call-to-action/demo/), `<svg>` or `<pf-icons>` should be spaced the same using `var(--rh-space-md, 8px)`
